### PR TITLE
🌐 Add Russian translation for `docs/ru/docs/tutorial/handling-errors.md`

### PR DESCRIPTION
**Pull request by AlertRED**

### CODE REVIEW

**Comment by tiangolo** [comment]
📝 Docs preview for commit 0b3fc0e941b8c9938b926ba0ad4d15d8b7b39491 at: https://3908cccf.fastapitiangolo.pages.dev

File: docs/ru/docs/tutorial/handling-errors.md [hunk 1]
```diff
@@ -0,0 +1,261 @@
...
+
+### Переиспользование обработчиков исключений **FastAPI**
+
+Если вы хотите использовать исключение вместе с теми же обработчиками исключений по умолчанию из **FastAPI**, вы можете импортировать и повторно использовать обработчики исключений по умолчанию из `fastapi.exception_handlers`:
+
+```Python hl_lines="2-5  15  21"
+{!../../../docs_src/handling_errors/tutorial006.py!}
+```
+
+В этом примере вы просто `выводите` ошибку с очень выразительным сообщением, но идея вам понятна. Вы можете использовать исключение, а затем просто повторно использовать стандартные обработчики исключений.
```
Comment (Xewus) [inline docs/ru/docs/tutorial/handling-errors.md:261]:
```suggestion
В этом примере вы просто `выводите в терминал` ошибку с очень выразительным сообщением, но идея вам понятна. Вы можете использовать исключение, а затем просто повторно использовать стандартные обработчики исключений.
```

**Review by Xewus — APPROVED** [review]

**Comment by tiangolo** [comment]
📝 Docs preview for commit 56b27fea553358c8aea05cc0893c5a1a01d903c2 at: https://c11593c8.fastapitiangolo.pages.dev

**Comment by alejsdev** [comment]
Thanks! @AlertRED 😃🏅 

And thanks for the help @Xewus 🎉

**Pull request merged**